docs/docs/clients/server-side.mdx

```diff
@@ -367,12 +371,12 @@ var buttonData = await flags.GetFeatureValue("secret_button");
 ```
 
 </TabItem>
-<TabItem value="nodejs" label="NodeJS">
+<TabItem value="nodejs" label="Node.js">
 
 ```javascript
 const flags = await flagsmith.getEnvironmentFlags();
-var showButton = flags.isFeatureEnabled('secret_button');
-var buttonData = flags.getFeatureValue('secret_button');
+const showButton = flags.isFeatureEnabled('secret_button');
+const buttonData = flags.getFeatureValue('secret_button');
```

Comment:
If we're changing this here, we should also do it [here](https://github.com/Flagsmith/flagsmith/pull/4821/files#diff-5e7e3806c40333115b787651c4deddb173a521ce8c3e6d2777a68c1652e169e1R493-R494) for consistency.